<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="classes" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="com.projectName.companyName.ExtentListeners.ExtentListeners"/>
    <listener class-name="com.projectName.companyName.helper.listener.RetryListener"/>
  </listeners>
  <test thread-count="5" parallel="classes" name="Test1(failed)">
    <classes>
      <class name="com.projectName.companyName.testcases.ActiveMembersLoginTest7">
        <methods>
          <include name="mail"/>
          <include name="setUpFramework"/>
          <include name="tearDown"/>
          <include name="loginTest" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.projectName.companyName.testcases.ActiveMembersLoginTest7 -->
    </classes>
  </test> <!-- Test1(failed) -->
</suite> <!-- Failed suite [Suite] -->
